package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "node-sass",
   "version": "3.3.2",
-  "libsass": "3.2.5",
+  "libsass": "3.3.0",
```

Comment:
On Wed, 16 Sep 2015, Michael Mifsud wrote:

> > @@ -1,7 +1,7 @@
> >  {
> >    "name": "node-sass",
> >    "version": "3.3.2",
> > -  "libsass": "3.2.5",
> > +  "libsass": "3.3.0",
> 
> `3.3.0-beta1`

We are already one merge above :)
We can adjust as we are close to merge this.